modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationService.java

```diff
@@ -234,4 +237,28 @@ public long getProgressTotal() {
      */
     void runStep(String id, String step);
 
+    /**
+     * @since 11.1
+     */
+    Migration getMigration(String id);
+
+    /**
+     * @since 11.1
+     */
+    List<Migration> getMigrations();
+
+    /**
+     * @throws IllegalStateException if the migration doesn't have exactly one available step to run.
+     * @since 11.1
+     */
+    void run(String migrationId) throws IllegalArgumentException;
+
+    /**
+     * Waits for completion of all migration steps.
+     *
+     * @return {@code true} if the migration executor has no more threads running {@code false} otherwise
```

Comment:
added some html tags and extra infos: ```/**
     * Waits for completion of all migration steps.<br/>
     * <b>ONLY FOR TEST PURPOSE</b>
     * <ul>
     * <li>Not suitable for use in a cluster environment.</li>
     * <li>Not declared in the API.</li>
     * </ul>
     *
     * @return {@code true} if the migration executor has no more threads running, {@code false} otherwise
     * @since 11.1
     */```